src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1284,10 +1284,6 @@ void TorrentHandle::moveStorage(const QString &newPath)
         if (QDir(oldPath) == QDir(newPath)) return;
 
         qDebug("move storage: %s to %s", qPrintable(oldPath), qPrintable(newPath));
-        // Create destination directory if necessary
-        // or move_storage() will fail...
```

Comment:
Yes. Besides, I deal with `move_storage()` internals recently so I know exactly how it works.